tests/integration_tests_control_files/advection_explicit_periodic/CMakeLists.txt

```diff
@@ -13,6 +13,14 @@ COMMAND mpirun -n ${MPIMAX} ${EXECUTABLE_OUTPUT_PATH}/PHiLiP_2D -i ${CMAKE_CURRE
   WORKING_DIRECTORY ${TEST_OUTPUT_DIR}
 )
 
+set(TEST_OUTPUT_DIR ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
I think that organizationally, it would make more sense to group the ESFR c tests in their own directory, rather than with the tests you've built  them from. This is just my opinion; maybe others can weigh in.